### PR TITLE
Update mesmerizing_dose.txt

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/mesmerizing_dose.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ Name:Mesmerizing Dose
 ManaCost:1 U U
 Types:Enchantment Aura
 K:Enchant creature
-A:SP$ Attach | Cost$ 1 B | ValidTgts$ Creature.YouCtrl | AITgts$ Card.nonToken | AILogic$ Pump
+A:SP$ Attach | Cost$ 1 B | ValidTgts$ Creature | AITgts$ Card.nonToken | AILogic$ Pump
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/mesmerizing_dose.txt:5]: that AILogic seems rather wrong?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/mesmerizing_dose.txt]: Still there...

File: forge-gui/res/cardsfolder/upcoming/mesmerizing_dose.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@ Name:Mesmerizing Dose
 ManaCost:1 U U
 Types:Enchantment Aura
 K:Enchant creature
-A:SP$ Attach | Cost$ 1 B | ValidTgts$ Creature.YouCtrl | AITgts$ Card.nonToken | AILogic$ Pump
+A:SP$ Attach | Cost$ 1 B | ValidTgts$ Creature
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/mesmerizing_dose.txt:5]: still wrong cost

File: forge-gui/res/cardsfolder/upcoming/mesmerizing_dose.txt [hunk 3]
```diff
@@ -2,7 +2,7 @@ Name:Mesmerizing Dose
 ManaCost:1 U U
 Types:Enchantment Aura
 K:Enchant creature
-A:SP$ Attach | Cost$ 1 B | ValidTgts$ Creature.YouCtrl | AITgts$ Card.nonToken | AILogic$ Pump
+A:SP$ Attach | Cost$ 1 U U | ValidTgts$ Creature
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/mesmerizing_dose.txt:5]: Removing something isn't always the right fix, so it's best to check if unsure
in this case the AI would no longer play it
you should use `AILogic$ KeepTapped`

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/mesmerizing_dose.txt]: Thanks for the explanation will keep in mind

**Review by tool4ever — APPROVED** [review]

**Pull request merged**